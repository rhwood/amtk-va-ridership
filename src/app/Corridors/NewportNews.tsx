import * as React from 'react';
import { PageSection } from '@patternfly/react-core/dist/dynamic/components/Page'
import { Title } from '@patternfly/react-core/dist/dynamic/components/Title'
import { MultiYearMonthlyOTPChart, MultiYearMonthlyRidershipChart, VPRAStats } from '@app/utils/MultiYearChart'
import { TextContent, TextList, TextListItem, TextVariants } from '@patternfly/react-core/dist/dynamic/components/Text'

const NewportNews: React.FunctionComponent = () => (
  <PageSection>
    <Title headingLevel="h1" size="lg">Newport News Corridor</Title>
    <TextContent>
      <Title headingLevel="h3">Ridership</Title>
      <MultiYearMonthlyRidershipChart lineFn={(row: VPRAStats): number => row['Newport News']} />
      <Title headingLevel="h3">On Time Performance</Title>
      <MultiYearMonthlyOTPChart lineFn={(row: VPRAStats): number => row['Newport News']} />
      <Title headingLevel={TextVariants.h3}>Notes</Title>
      <TextList isPlain>
        <TextListItem>A roundtrip cancelled due to the pandemic was resumed in July 2022.</TextListItem>
      </TextList>
    </TextContent>
  </PageSection>
)

export { NewportNews };
