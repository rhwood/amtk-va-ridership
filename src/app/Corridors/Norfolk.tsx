import * as React from 'react';
import { PageSection } from '@patternfly/react-core/dist/dynamic/components/Page'
import { Title } from '@patternfly/react-core/dist/dynamic/components/Title'
import { MultiYearMonthlyOTPChart, MultiYearMonthlyRidershipChart, VPRAStats } from '@app/utils/MultiYearChart'
import { TextContent, TextList, TextListItem, TextVariants } from '@patternfly/react-core/dist/dynamic/components/Text'

const Norfolk: React.FunctionComponent = () => (
  <PageSection>
    <Title headingLevel="h1" size="lg">Norfolk Corridor</Title>
    <TextContent>
      <Title headingLevel={TextVariants.h3}>Ridership</Title>
      <MultiYearMonthlyRidershipChart lineFn={(row: VPRAStats): number => row.Norfolk} />
      <Title headingLevel={TextVariants.h3}>On Time Performance</Title>
      <MultiYearMonthlyOTPChart lineFn={(row: VPRAStats): number => row.Norfolk} />
      <Title headingLevel={TextVariants.h3}>Notes</Title>
      <TextList isPlain>
        <TextListItem>A second roundtrip was added in March 2019.</TextListItem>
        <TextListItem>A third roundtrip was added in July 2022.</TextListItem>
      </TextList>
    </TextContent>
  </PageSection>
)

export { Norfolk };
