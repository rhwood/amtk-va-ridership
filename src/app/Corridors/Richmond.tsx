import * as React from 'react';
import { PageSection } from '@patternfly/react-core/dist/dynamic/components/Page'
import { Title } from '@patternfly/react-core/dist/dynamic/components/Title'
import { MultiYearChart, Ridership } from '@app/utils/MultiYearChart';
import { Text, TextContent, TextList, TextListItem, TextVariants } from '@patternfly/react-core/dist/dynamic/components/Text';

const Richmond: React.FunctionComponent = () => (
  <PageSection>
    <Title headingLevel="h1" size="lg">Richmond Corridor Ridership</Title>
    <TextContent>
      <Text component={TextVariants.p}>
        The Richmond corridor consists of Northeast Regional services terminating at Richmond Main Street station.
        The entirety of the Richmond corridor is shared with the Newport News corridor, and all of the Richmod
        corridor except the Richmond Main Street station is shared with the Norfolk corridor.
      </Text>
    </TextContent>
    <MultiYearChart lineFn={(row: Ridership): number => row.Richmond} />
    <TextContent>
      <Text component={TextVariants.h3}>Notes</Text>
      <TextList isPlain>
        <TextListItem>Services only on this corridor was suspended from March 2020 to August 2021.</TextListItem>
        <TextListItem>Ridership origininating or terminating at stations in this corridor, but on Newport News
          or Norfolk corridor services are shown under those services.</TextListItem>
      </TextList>
    </TextContent>
  </PageSection>
)

export { Richmond };
