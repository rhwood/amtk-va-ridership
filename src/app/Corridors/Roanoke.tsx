import * as React from 'react';
import { PageSection } from '@patternfly/react-core/dist/dynamic/components/Page'
import { Title } from '@patternfly/react-core/dist/dynamic/components/Title'
import { MultiYearMonthlyChart, Ridership } from '@app/utils/MultiYearChart';
import { Text, TextContent, TextList, TextListItem, TextVariants } from '@patternfly/react-core/dist/dynamic/components/Text';

const Roanoke: React.FunctionComponent = () => (
  <PageSection>
    <Title headingLevel="h1" size="lg">Roanoke Corridor Ridership</Title>
    <TextContent>
      <MultiYearMonthlyChart lineFn={(row: Ridership): number => row.Roanoke} />
      <Text component={TextVariants.h3}>Notes</Text>
      <TextList isPlain>
        <TextListItem>A second roundtrip was added in July 2022.</TextListItem>
      </TextList>
    </TextContent>
  </PageSection>
)

export { Roanoke };
