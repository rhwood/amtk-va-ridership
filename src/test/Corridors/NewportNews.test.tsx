import * as React from 'react';
import { NewportNews } from '@app/Corridors/NewportNews';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom'

describe('NewportNews tests', () => {
    test('should render NewportNews component', () => {
        const { asFragment } = render(<NewportNews />);
  
        expect(asFragment()).toMatchSnapshot();
    });
});
