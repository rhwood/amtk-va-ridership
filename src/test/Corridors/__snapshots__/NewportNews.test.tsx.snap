// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewportNews tests should render NewportNews component 1`] = `
<DocumentFragment>
  <section
    class="pf-v5-c-page__main-section"
  >
    <h1
      class="pf-v5-c-title pf-m-lg"
      data-ouia-component-id="OUIA-Generated-Title-1"
      data-ouia-component-type="PF5/Title"
      data-ouia-safe="true"
    >
      Newport News Corridor Ridership
    </h1>
    <div
      class="pf-v5-c-chart"
      style="height: 100%; width: 100%; user-select: none; pointer-events: none; position: relative;"
    >
      <svg
        height="250"
        role="img"
        style="pointer-events: all; width: 100%; height: 100%;"
        viewBox="0 0 600 250"
        width="600"
      >
        <g
          role="presentation"
        >
          <line
            role="presentation"
            shape-rendering="auto"
            style="stroke: var(--pf-v5-chart-axis--axis--stroke--Color, #d2d2d2); fill: var(--pf-v5-chart-axis--axis--Fill, transparent); stroke-width: 1; stroke-linecap: round; stroke-linejoin: round;"
            vector-effect="non-scaling-stroke"
            x1="50"
            x2="400"
            y1="200"
            y2="200"
          />
          <g
            role="presentation"
          >
            <line
              role="presentation"
              shape-rendering="auto"
              style="stroke: var(--pf-v5-chart-axis--tick--stroke--Color, #d2d2d2); fill: var(--pf-v5-chart-axis--tick--Fill, transparent); size: 5px; stroke-linecap: round; stroke-linejoin: round; stroke-width: 1;"
              vector-effect="non-scaling-stroke"
              x1="50"
              x2="50"
              y1="200"
              y2="205"
            />
            <text
              direction="inherit"
              dx="0"
              id="Total Ridership-ChartAxis-0-ChartLabel-0"
              x="50"
              y="226.97"
            >
              <tspan
                dx="0"
                dy="0"
                style="font-family: "RedHatText",helvetica,arial,sans-serif; font-size: 14px; letter-spacing: normal; padding: 10px; stroke: var(--pf-v5-chart-global--label--stroke, transparent); fill: var(--pf-v5-chart-axis--tick-label--Fill, #4f5255); text-anchor: middle;"
                text-anchor="middle"
                x="50"
              >
                Jan
              </tspan>
            </text>
          </g>
          <g
            role="presentation"
          >
            <line
              role="presentation"
              shape-rendering="auto"
              style="stroke: var(--pf-v5-chart-axis--tick--stroke--Color, #d2d2d2); fill: var(--pf-v5-chart-axis--tick--Fill, transparent); size: 5px; stroke-linecap: round; stroke-linejoin: round; stroke-width: 1;"
              vector-effect="non-scaling-stroke"
              x1="81.81818181818181"
              x2="81.81818181818181"
              y1="200"
              y2="205"
            />
            <text
              direction="inherit"
              dx="0"
              id="Total Ridership-ChartAxis-0-ChartLabel-1"
              x="81.81818181818181"
              y="226.97"
            >
              <tspan
                dx="0"
                dy="0"
                style="font-family: "RedHatText",helvetica,arial,sans-serif; font-size: 14px; letter-spacing: normal; padding: 10px; stroke: var(--pf-v5-chart-global--label--stroke, transparent); fill: var(--pf-v5-chart-axis--tick-label--Fill, #4f5255); text-anchor: middle;"
                text-anchor="middle"
                x="81.81818181818181"
              >
                Feb
              </tspan>
            </text>
          </g>
          <g
            role="presentation"
          >
            <line
              role="presentation"
              shape-rendering="auto"
              style="stroke: var(--pf-v5-chart-axis--tick--stroke--Color, #d2d2d2); fill: var(--pf-v5-chart-axis--tick--Fill, transparent); size: 5px; stroke-linecap: round; stroke-linejoin: round; stroke-width: 1;"
              vector-effect="non-scaling-stroke"
              x1="113.63636363636364"
              x2="113.63636363636364"
              y1="200"
              y2="205"
            />
            <text
              direction="inherit"
              dx="0"
              id="Total Ridership-ChartAxis-0-ChartLabel-2"
              x="113.63636363636364"
              y="226.97"
            >
              <tspan
                dx="0"
                dy="0"
                style="font-family: "RedHatText",helvetica,arial,sans-serif; font-size: 14px; letter-spacing: normal; padding: 10px; stroke: var(--pf-v5-chart-global--label--stroke, transparent); fill: var(--pf-v5-chart-axis--tick-label--Fill, #4f5255); text-anchor: middle;"
                text-anchor="middle"
                x="113.63636363636364"
              >
                Mar
              </tspan>
            </text>
          </g>
          <g
            role="presentation"
          >
            <line
              role="presentation"
              shape-rendering="auto"
              style="stroke: var(--pf-v5-chart-axis--tick--stroke--Color, #d2d2d2); fill: var(--pf-v5-chart-axis--tick--Fill, transparent); size: 5px; stroke-linecap: round; stroke-linejoin: round; stroke-width: 1;"
              vector-effect="non-scaling-stroke"
              x1="145.45454545454544"
              x2="145.45454545454544"
              y1="200"
              y2="205"
            />
            <text
              direction="inherit"
              dx="0"
              id="Total Ridership-ChartAxis-0-ChartLabel-3"
              x="145.45454545454544"
              y="226.97"
            >
              <tspan
                dx="0"
                dy="0"
                style="font-family: "RedHatText",helvetica,arial,sans-serif; font-size: 14px; letter-spacing: normal; padding: 10px; stroke: var(--pf-v5-chart-global--label--stroke, transparent); fill: var(--pf-v5-chart-axis--tick-label--Fill, #4f5255); text-anchor: middle;"
                text-anchor="middle"
                x="145.45454545454544"
              >
                Apr
              </tspan>
            </text>
          </g>
          <g
            role="presentation"
          >
            <line
              role="presentation"
              shape-rendering="auto"
              style="stroke: var(--pf-v5-chart-axis--tick--stroke--Color, #d2d2d2); fill: var(--pf-v5-chart-axis--tick--Fill, transparent); size: 5px; stroke-linecap: round; stroke-linejoin: round; stroke-width: 1;"
              vector-effect="non-scaling-stroke"
              x1="177.27272727272728"
              x2="177.27272727272728"
              y1="200"
              y2="205"
            />
            <text
              direction="inherit"
              dx="0"
              id="Total Ridership-ChartAxis-0-ChartLabel-4"
              x="177.27272727272728"
              y="226.97"
            >
              <tspan
                dx="0"
                dy="0"
                style="font-family: "RedHatText",helvetica,arial,sans-serif; font-size: 14px; letter-spacing: normal; padding: 10px; stroke: var(--pf-v5-chart-global--label--stroke, transparent); fill: var(--pf-v5-chart-axis--tick-label--Fill, #4f5255); text-anchor: middle;"
                text-anchor="middle"
                x="177.27272727272728"
              >
                May
              </tspan>
            </text>
          </g>
          <g
            role="presentation"
          >
            <line
              role="presentation"
              shape-rendering="auto"
              style="stroke: var(--pf-v5-chart-axis--tick--stroke--Color, #d2d2d2); fill: var(--pf-v5-chart-axis--tick--Fill, transparent); size: 5px; stroke-linecap: round; stroke-linejoin: round; stroke-width: 1;"
              vector-effect="non-scaling-stroke"
              x1="209.0909090909091"
              x2="209.0909090909091"
              y1="200"
              y2="205"
            />
            <text
              direction="inherit"
              dx="0"
              id="Total Ridership-ChartAxis-0-ChartLabel-5"
              x="209.0909090909091"
              y="226.97"
            >
              <tspan
                dx="0"
                dy="0"
                style="font-family: "RedHatText",helvetica,arial,sans-serif; font-size: 14px; letter-spacing: normal; padding: 10px; stroke: var(--pf-v5-chart-global--label--stroke, transparent); fill: var(--pf-v5-chart-axis--tick-label--Fill, #4f5255); text-anchor: middle;"
                text-anchor="middle"
                x="209.0909090909091"
              >
                Jun
              </tspan>
            </text>
          </g>
          <g
            role="presentation"
          >
            <line
              role="presentation"
              shape-rendering="auto"
              style="stroke: var(--pf-v5-chart-axis--tick--stroke--Color, #d2d2d2); fill: var(--pf-v5-chart-axis--tick--Fill, transparent); size: 5px; stroke-linecap: round; stroke-linejoin: round; stroke-width: 1;"
              vector-effect="non-scaling-stroke"
              x1="240.90909090909088"
              x2="240.90909090909088"
              y1="200"
              y2="205"
            />
            <text
              direction="inherit"
              dx="0"
              id="Total Ridership-ChartAxis-0-ChartLabel-6"
              x="240.90909090909088"
              y="226.97"
            >
              <tspan
                dx="0"
                dy="0"
                style="font-family: "RedHatText",helvetica,arial,sans-serif; font-size: 14px; letter-spacing: normal; padding: 10px; stroke: var(--pf-v5-chart-global--label--stroke, transparent); fill: var(--pf-v5-chart-axis--tick-label--Fill, #4f5255); text-anchor: middle;"
                text-anchor="middle"
                x="240.90909090909088"
              >
                Jul
              </tspan>
            </text>
          </g>
          <g
            role="presentation"
          >
            <line
              role="presentation"
              shape-rendering="auto"
              style="stroke: var(--pf-v5-chart-axis--tick--stroke--Color, #d2d2d2); fill: var(--pf-v5-chart-axis--tick--Fill, transparent); size: 5px; stroke-linecap: round; stroke-linejoin: round; stroke-width: 1;"
              vector-effect="non-scaling-stroke"
              x1="272.7272727272727"
              x2="272.7272727272727"
              y1="200"
              y2="205"
            />
            <text
              direction="inherit"
              dx="0"
              id="Total Ridership-ChartAxis-0-ChartLabel-7"
              x="272.7272727272727"
              y="226.97"
            >
              <tspan
                dx="0"
                dy="0"
                style="font-family: "RedHatText",helvetica,arial,sans-serif; font-size: 14px; letter-spacing: normal; padding: 10px; stroke: var(--pf-v5-chart-global--label--stroke, transparent); fill: var(--pf-v5-chart-axis--tick-label--Fill, #4f5255); text-anchor: middle;"
                text-anchor="middle"
                x="272.7272727272727"
              >
                Aug
              </tspan>
            </text>
          </g>
          <g
            role="presentation"
          >
            <line
              role="presentation"
              shape-rendering="auto"
              style="stroke: var(--pf-v5-chart-axis--tick--stroke--Color, #d2d2d2); fill: var(--pf-v5-chart-axis--tick--Fill, transparent); size: 5px; stroke-linecap: round; stroke-linejoin: round; stroke-width: 1;"
              vector-effect="non-scaling-stroke"
              x1="304.54545454545456"
              x2="304.54545454545456"
              y1="200"
              y2="205"
            />
            <text
              direction="inherit"
              dx="0"
              id="Total Ridership-ChartAxis-0-ChartLabel-8"
              x="304.54545454545456"
              y="226.97"
            >
              <tspan
                dx="0"
                dy="0"
                style="font-family: "RedHatText",helvetica,arial,sans-serif; font-size: 14px; letter-spacing: normal; padding: 10px; stroke: var(--pf-v5-chart-global--label--stroke, transparent); fill: var(--pf-v5-chart-axis--tick-label--Fill, #4f5255); text-anchor: middle;"
                text-anchor="middle"
                x="304.54545454545456"
              >
                Sep
              </tspan>
            </text>
          </g>
          <g
            role="presentation"
          >
            <line
              role="presentation"
              shape-rendering="auto"
              style="stroke: var(--pf-v5-chart-axis--tick--stroke--Color, #d2d2d2); fill: var(--pf-v5-chart-axis--tick--Fill, transparent); size: 5px; stroke-linecap: round; stroke-linejoin: round; stroke-width: 1;"
              vector-effect="non-scaling-stroke"
              x1="336.3636363636364"
              x2="336.3636363636364"
              y1="200"
              y2="205"
            />
            <text
              direction="inherit"
              dx="0"
              id="Total Ridership-ChartAxis-0-ChartLabel-9"
              x="336.3636363636364"
              y="226.97"
            >
              <tspan
                dx="0"
                dy="0"
                style="font-family: "RedHatText",helvetica,arial,sans-serif; font-size: 14px; letter-spacing: normal; padding: 10px; stroke: var(--pf-v5-chart-global--label--stroke, transparent); fill: var(--pf-v5-chart-axis--tick-label--Fill, #4f5255); text-anchor: middle;"
                text-anchor="middle"
                x="336.3636363636364"
              >
                Oct
              </tspan>
            </text>
          </g>
          <g
            role="presentation"
          >
            <line
              role="presentation"
              shape-rendering="auto"
              style="stroke: var(--pf-v5-chart-axis--tick--stroke--Color, #d2d2d2); fill: var(--pf-v5-chart-axis--tick--Fill, transparent); size: 5px; stroke-linecap: round; stroke-linejoin: round; stroke-width: 1;"
              vector-effect="non-scaling-stroke"
              x1="368.1818181818182"
              x2="368.1818181818182"
              y1="200"
              y2="205"
            />
            <text
              direction="inherit"
              dx="0"
              id="Total Ridership-ChartAxis-0-ChartLabel-10"
              x="368.1818181818182"
              y="226.97"
            >
              <tspan
                dx="0"
                dy="0"
                style="font-family: "RedHatText",helvetica,arial,sans-serif; font-size: 14px; letter-spacing: normal; padding: 10px; stroke: var(--pf-v5-chart-global--label--stroke, transparent); fill: var(--pf-v5-chart-axis--tick-label--Fill, #4f5255); text-anchor: middle;"
                text-anchor="middle"
                x="368.1818181818182"
              >
                Nov
              </tspan>
            </text>
          </g>
          <g
            role="presentation"
          >
            <line
              role="presentation"
              shape-rendering="auto"
              style="stroke: var(--pf-v5-chart-axis--tick--stroke--Color, #d2d2d2); fill: var(--pf-v5-chart-axis--tick--Fill, transparent); size: 5px; stroke-linecap: round; stroke-linejoin: round; stroke-width: 1;"
              vector-effect="non-scaling-stroke"
              x1="400"
              x2="400"
              y1="200"
              y2="205"
            />
            <text
              direction="inherit"
              dx="0"
              id="Total Ridership-ChartAxis-0-ChartLabel-11"
              x="400"
              y="226.97"
            >
              <tspan
                dx="0"
                dy="0"
                style="font-family: "RedHatText",helvetica,arial,sans-serif; font-size: 14px; letter-spacing: normal; padding: 10px; stroke: var(--pf-v5-chart-global--label--stroke, transparent); fill: var(--pf-v5-chart-axis--tick-label--Fill, #4f5255); text-anchor: middle;"
                text-anchor="middle"
                x="400"
              >
                Dec
              </tspan>
            </text>
          </g>
        </g>
        <g
          role="presentation"
        >
          <line
            role="presentation"
            shape-rendering="auto"
            style="stroke: var(--pf-v5-chart-axis--axis--stroke--Color, #d2d2d2); fill: var(--pf-v5-chart-axis--axis--Fill, transparent); stroke-width: 1; stroke-linecap: round; stroke-linejoin: round;"
            vector-effect="non-scaling-stroke"
            x1="50"
            x2="50"
            y1="50"
            y2="200"
          />
          <g
            role="presentation"
          >
            <line
              role="presentation"
              shape-rendering="auto"
              style="stroke: var(--pf-v5-chart-axis--grid--stroke--Color, #d2d2d2); fill: var(--pf-v5-chart-axis--grid--Fill, none); pointer-events: painted; stroke-linecap: round; stroke-linejoin: round;"
              vector-effect="non-scaling-stroke"
              x1="50"
              x2="400"
              y1="200"
              y2="200"
            />
            <line
              role="presentation"
              shape-rendering="auto"
              style="stroke: var(--pf-v5-chart-axis--tick--stroke--Color, #d2d2d2); fill: var(--pf-v5-chart-axis--tick--Fill, transparent); size: 5px; stroke-linecap: round; stroke-linejoin: round; stroke-width: 1;"
              vector-effect="non-scaling-stroke"
              x1="50"
              x2="45"
              y1="200"
              y2="200"
            />
            <text
              direction="inherit"
              dx="0"
              id="Total Ridership-ChartAxis-1-ChartLabel-0"
              x="35"
              y="204.97"
            >
              <tspan
                dx="0"
                dy="0"
                style="font-family: "RedHatText",helvetica,arial,sans-serif; font-size: 14px; letter-spacing: normal; padding: 10px; stroke: var(--pf-v5-chart-global--label--stroke, transparent); fill: var(--pf-v5-chart-axis--tick-label--Fill, #4f5255); text-anchor: end;"
                text-anchor="end"
                x="35"
              />
            </text>
          </g>
        </g>
        <g>
          <g
            clip-path="url(#victory-clip-2)"
            style="height: 100%; width: 100%; user-select: none;"
          >
            <defs>
              <clippath
                id="victory-clip-2"
              >
                <rect
                  height="150"
                  vector-effect="non-scaling-stroke"
                  width="350"
                  x="50"
                  y="50"
                />
              </clippath>
            </defs>
            <path
              d="M113.63636363636364,50L145.45454545454544,50"
              role="presentation"
              shape-rendering="auto"
              style="fill: none; stroke: var(--pf-v5-chart-theme--multi-color-unordered--ColorScale--100, #06c); pointer-events: stroke; width: 88px; opacity: 1; stroke-width: 2;"
            />
          </g>
        </g>
        <g>
          <rect
            height="14"
            role="presentation"
            shape-rendering="auto"
            style="fill: none;"
            vector-effect="non-scaling-stroke"
            width="39.6"
            x="416"
            y="120"
          />
          <path
            d="M 425.128, 138.872
      h9.744000000000028
      v-9.744000000000028
      h-9.744000000000028
      z"
            role="presentation"
            shape-rendering="auto"
            style="fill: var(--pf-v5-chart-theme--multi-color-unordered--ColorScale--100, #06c);"
          />
          <text
            direction="inherit"
            dx="0"
            id="Total Ridership-ChartLegend-ChartLabel-0"
            x="446.8"
            y="138.97"
          >
            <tspan
              dx="0"
              dy="0"
              style="font-family: "RedHatText",helvetica,arial,sans-serif; font-size: 14px; letter-spacing: normal; padding: 10px; stroke: var(--pf-v5-chart-global--label--stroke, transparent); fill: var(--pf-v5-chart-global--label--Fill, #151515);"
              text-anchor="start"
              x="446.8"
            >
              1900
            </tspan>
          </text>
        </g>
      </svg>
      <div
        style="z-index: 99; position: absolute; top: 0px; left: 0px; width: 100%; height: 100%;"
      >
        <svg
          height="250"
          style="overflow: visible; width: 100%; height: 100%;"
          viewBox="0 0 600 250"
          width="600"
        />
      </div>
    </div>
    <div
      class="pf-v5-c-content"
    >
      <h3
        class=""
        data-ouia-component-id="OUIA-Generated-Text-1"
        data-ouia-component-type="PF5/Text"
        data-ouia-safe="true"
        data-pf-content="true"
      >
        Notes
      </h3>
      <ul
        class="pf-m-plain"
      >
        <li
          class=""
        >
          A roundtrip cancelled due to the pandemic was resumed in July 2022.
        </li>
      </ul>
    </div>
  </section>
</DocumentFragment>
`;
